modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java

```diff
@@ -1,8 +1,8 @@
 /*
- * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com).
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why we need to change the license?

Reply:
Addressed in 775ef66a9ca19b6e1300ff29e7edfaf856ce7d12